centreon/tests/e2e/cypress/e2e/Resources-status/04-timezone/index.ts

```diff
@@ -0,0 +1,382 @@
+import { Given, Then, When } from '@badeball/cypress-cucumber-preprocessor';
+
+import {
+  checkServicesAreMonitored,
+  submitResultsViaClapi,
+  updateFixturesResult
+} from '../../../commons';
+import {
+  actionBackgroundColors,
+  insertDtResources,
+  secondServiceInDtName,
+  serviceInAcknowledgementName,
+  serviceInDtName,
+  tearDownResource
+} from '../common';
+
+const chosenTZ = 'Africa/Casablanca';
+
+const extract12hFormatFromTimeString = (dateTime: string): string => {
+  const timeRegex = /(\d{1,2}:\d{2} [AP]M)$/;
+
+  const extractedTime = dateTime?.match(timeRegex);
+
+  return extractedTime ? extractedTime[0] : '';
+};
+
+before(() => {
+  cy.startWebContainer();
+});
+
+beforeEach(() => {
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/internal.php?object=centreon_topology&action=navigationList'
+  }).as('getNavigationList');
+  cy.intercept({
+    method: 'POST',
+    url: '/centreon/api/latest/monitoring/resources/downtime'
+  }).as('postSaveDowntime');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/include/common/webServices/rest/internal.php?object=centreon_configuration_timezone&action=list*'
+  }).as('getTimezonesList');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/latest/monitoring/hosts/*/services/*/acknowledgements?limit=1'
+  }).as('getAckToolTip');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/internal.php?object=centreon_configuration_service&action=list&e=enable&page_limit=60&page=1'
+  }).as('getServices');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/include/common/userTimezone.php'
+  }).as('getTimeZone');
+  cy.intercept({
+    method: 'POST',
+    url: '/centreon/api/latest/monitoring/resources/acknowledge'
+  }).as('postAcknowledgments');
+});
+
+Given('a user authenticated in a Centreon server', () => {
+  cy.loginByTypeOfUser({
+    jsonName: 'admin',
+    loginViaApi: true
+  });
+});
+
+Given('the platform is configured with at least one resource', () => {
+  cy.reload();
+
+  insertDtResources();
```

Comment:
The fixture insertion will need to be refactored
Because all tests insert DtResources or AckResources and it's hard to understand which service and host name are used in the test
can be handled in an other PR

Reply:
let's write it down and fix it asap then

Reply:
Maybe host and services should be generic. Like ok, we insert some resources. Each tests will determine what we do for those resources